api/src/test/resources/io/strimzi/api/kafka/model/KafkaAssembly-with-invalid-enums.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: kafka.strimzi.io/v1alpha1
```

Comment:
If this is just testing invalid enums to don't need all the resources, tolerations etc.